linkerd/app/inbound/src/policy/http.rs

```diff
@@ -65,6 +65,20 @@ pub struct HttpRouteRedirect {
 #[error("unauthorized request on route")]
 pub struct HttpRouteUnauthorized(());
 
+#[derive(Debug, thiserror::Error, Clone, PartialEq)]
```

Comment:
nit, not important: i _think_ we can still import `thiserror::Error` even though `linkerd_app_core::Error` is also in scope; IIRC derive macros occupy a different namespace from traits.

Reply:
it's done this way elsewhere in the file already. would rather keep the change small/consistent for now.